gulpfile.js

```diff
@@ -65,73 +58,11 @@ function runCommand(command) {
 
 const WEBAPP_OUT = 'www';
 
-// Copies Babel polyfill from node_modules, as it needs to be included by cordova_index.html.
-function copyBabelPolyfill() {
-  const babelPolyfill = 'node_modules/babel-polyfill/dist/polyfill.min.js';
-  return runCommand(`cp -v ${babelPolyfill} ${WEBAPP_OUT}/babel-polyfill.min.js`);
-}
-
-// Bundles code with the entry point www/app/cordova_main.js -> www/cordova_main.js.
-//
-// Useful Gulp/Browserify examples:
-//   https://github.com/gulpjs/gulp/tree/master/docs/recipes
-function browserifyAndBabelify() {
-  return browserify({entries: `${WEBAPP_OUT}/app/cordova_main.js`, debug: true})
-      .transform('babelify', {
-        // Transpile code in node_modules, too.
-        global: true,
-        presets: ['env']
-      })
-      .bundle()
-      // Transform the bundle() output stream into one regular Gulp plugins understand.
-      .pipe(source('cordova_main.js'))
-      .pipe(gulp.dest(WEBAPP_OUT));
-}
-
-// Transpiles to |src| to ES5, copying the output to |dest|.
-function transpile(src, dest) {
-  const sourcesHtmlSplitter = new polymer_build.HtmlSplitter();
-  return gulp.src(src)
-      .pipe(sourcesHtmlSplitter.split())
-      .pipe(gulpif(/\.js$/, babel({presets: [babel_preset_env.default()]})))
-      .pipe(sourcesHtmlSplitter.rejoin())
-      .pipe(gulp.dest(dest));
-}
-
-// Note: This is currently done "in-place", i.e. the components are downloaded to `WEBAPP_OUT` and
-// transpiled there, but this seems to work just fine (idempodent).
-function transpileBowerComponents() {
-  // Transpile bower_components with the exception of webcomponentsjs, which contains transpiled
-  // polyfills, and minified files, which are generally already transpiled.
-  const bowerComponentsSrc = [
-    `${WEBAPP_OUT}/bower_components/**/*.html`, `${WEBAPP_OUT}/bower_components/**/*.js`,
-    `!${WEBAPP_OUT}/bower_components/webcomponentsjs/**/*.js`,
-    `!${WEBAPP_OUT}/bower_components/webcomponentsjs/**/*.html`,
-    `!${WEBAPP_OUT}/bower_components/**/*.min.js`
-  ];
-  const bowerComponentsDest = `${WEBAPP_OUT}/bower_components`;
-  return transpile(bowerComponentsSrc, bowerComponentsDest);
-}
-
-// Transpiling and generating RTL CSS happens sequentially, which means that the output of the first
-// step must be the source of the next; otherwise the last step will clobber the previous steps.
-// To avoid this, we transpile and generate RTL CSS in-place, with `WEBAPP_OUT` as input and output.
-function transpileUiComponents() {
-  return transpile([`${WEBAPP_OUT}/ui_components/*.html`], `${WEBAPP_OUT}/ui_components`);
-}
-
-function rtlCss() {
-  return generateRtlCss(`${WEBAPP_OUT}/ui_components/*.html`, `${WEBAPP_OUT}/ui_components`)
-}
-
 function buildWebApp() {
-  return runCommand(`yarn do src/www/build`);
+  const devMode = isRelease ? '' : 'export BUILD_ENV=development && ';
```

Comment:
Why are you building in DEV mode?
We shouldn't use dev in production

Reply:
I think the variable name I chose is confusing.

When I first converted to Webpack, all builds were running in `production` mode and there was no command line way to switch to `development` to enable features like source map. 

@alalamav suggested we could use the `--release` flag to switch mode for Cordova builds. So that could mean something like this:
```shell
yarn gulp build --platform=<platform>                # Webpack development mode
yarn gulp build --platform=<platform> --release      # Webpack production mode (plus productionization steps like app signing)
```

`devMode` only gets applied when the `isRelease` flag is not set. If we want to keep this behavior, I should really rename the variable.

Let me know what you think.

Reply:
The idea is to build for development if we're not releasing (the default is prod). Now that I think about it, maybe it would be safer to not export the variable, in case a subsequent build is prod - or always set the variable to the appropriate value.

Reply:
Good point. Maybe being explicit is a better approach.

Regarding subsequent builds, I'm not overly concerned. `export` sets the variable for the current and child processes, not the parent, so it should not affect subsequent builds.

Reply:
Added instructions to the README about building in development mode.

```sh
(export BUILD_ENV=development; yarn gulp build --platform=browser)
```